Documentation/prerequisites-for-building.md

```diff
@@ -12,12 +12,19 @@ PowerShell also needs to be available from the PATH environment variable (it's t
 
 Install basic dependency packages:
 
+First add a new package source to be able to install clang-3.9:
+```sh
+echo "deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.9 main" | sudo tee /etc/apt/sources.list.d/llvm.list
+wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
+sudo apt-get update
 ```
-sudo apt-get install cmake llvm-3.5 clang-3.5 lldb-3.6 lldb-3.6-dev liblttng-ust liblttng-ust-dev uuid uuid-dev
```

Comment:
Oh, right. I haven't realized that we are running coreclr during the build. But not all of them (the -dev ones are just development ones for building and not using coreclr, lldb is not needed). I can also see that the list is missing libunwind8, libcurl and libicu. I'll create a fresh docker container for Ubuntu 14.04 and figure out the real minimal set.